clients/python/scripts/pydantic.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+filename=$1
+
+while IFS= read -r line; do
+    echo "... $line ..."
+    modules=$(echo $line | awk -F 'from pydantic import ' '{print $2}')
+    echo $line
+    
+    newline="try:\n    from pydantic.v1 import $modules\nexcept ImportError:\n    from pydantic import $modules"
+    
+    echo "$newline"
+    
+    echo "Replacing import statement"
```

Comment:
I think all the `echo`s are great for debugging but kind of redundant, maybe just output the file